testng-core/src/main/java/org/testng/internal/ITestClassConfigInfo.java

```diff
@@ -15,6 +16,10 @@ default List<ITestNGMethod> getAllBeforeClassMethods() {
     return Lists.newArrayList();
   }
 
+  default List<ITestNGMethod> getAllAfterClassMethods() {
```

Comment:
As it is an internal interface, there is no need to provide a default dummy implementation.

Reply:
Fixed

Reply:
Just `List<ITestNGMethod> getAllAfterClassMethods();` please

Reply:
fixed